### PR TITLE
Update Welcome page and intoruce FAQ, About and Contacts pages

### PR DESCRIPTION
**Pull request by fidelman**
[Ticket](https://cesko-digital.atlassian.net/browse/JH-121)

# Done
- [x] Update text and links in the footer

- [x] Introduce FAQ page (both mobile and desktop)
screencapture-localhost-3000-2022-03-19-23_14_58
screencapture-localhost-3000-2022-03-19-23_15_50
screencapture-localhost-3000-2022-03-19-23_14_44
screencapture-localhost-3000-2022-03-19-23_15_24

- [x] Introduce Contacts page (both mobile and desktop)
screencapture-localhost-3000-2022-03-19-23_14_18
screencapture-localhost-3000-2022-03-19-23_16_16

- [x] Introduce About page (both mobile and desktop)
screencapture-localhost-3000-2022-03-19-23_13_37
screencapture-localhost-3000-2022-03-19-23_14_01
screencapture-localhost-3000-2022-03-19-23_13_50

### CODE REVIEW

File: web-jehlomat/src/Components/CollapsibleText/index.tsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import * as S from './styles';
+import { FC, useCallback, useState } from 'react';
+import { useMediaQuery } from '@mui/material';
+import { media } from 'utils/media';
+
+interface IProps {
+    title: string;
+    text: string; // could be HTML
```
Comment (Sebosek) [inline web-jehlomat/src/Components/CollapsibleText/index.tsx:8]: What would you say to define a `type HtmlText = string;`?

File: web-jehlomat/src/Components/CollapsibleText/index.tsx [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+}
+
+export const CollapsibleText: FC<IProps> = ({ title, text }) => {
+    const [open, setOpen] = useState(false);
+    const isMobile = useMediaQuery(media.lte('mobile'));
+
+    const handleClick = useCallback(() => {
+        setOpen(!open);
+    }, [open, setOpen]);
```
Comment (Sebosek) [inline web-jehlomat/src/Components/CollapsibleText/index.tsx:17]: I believe `setOpen` is not needed here.

**Review by Sebosek — APPROVED** [review]

File: web-jehlomat/src/screens/AboutPage/styles.ts [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+
+export const Title = styled(Typography)<{ isMobile: boolean }>`
+    color: ${primaryDark};
+    font-size: ${p => (p.isMobile ? '24px' : '48px')};
+    line-height: ${p => (p.isMobile ? '28px' : '56px')};
+    font-weight: 300px;
+    margin-top: ${p => (p.isMobile ? '30px' : '130px')};
+    margin-bottom: ${p => (p.isMobile ? '15px' : '0')};
+    width: ${p => (p.isMobile ? '100%' : 'auto')};
+`;
```
[the remark this answers is not in the document]
Comment (fidelman) [reply web-jehlomat/src/screens/AboutPage/styles.ts:19]: I think it is a kinda personal thing. I like personally when every line has its own condition.
- fewer lines
- less nesting
- one prop definition
I would leave my version and totally ok if someone uses another one in their code.

File: web-jehlomat/src/Components/CollapsibleText/index.tsx (hunk 2 above)
Comment (fidelman) [reply web-jehlomat/src/Components/CollapsibleText/index.tsx:17]: you are right

File: web-jehlomat/src/Components/CollapsibleText/index.tsx (hunk 1 above)
Comment (fidelman) [reply web-jehlomat/src/Components/CollapsibleText/index.tsx:8]: good

**Pull request merged**